apps/api/src/user/user.e2e.spec.ts

```diff
@@ -94,9 +95,28 @@ describe('User Controller Tests', () => {
         'x-e2e-user-email': regularUser.email
       }
     })
+
+    const workspace = await prisma.workspace.findFirst({
+      where: {
+        ownerId: regularUser.id,
+        isDefault: true
+      }
+    })
+
     expect(result.statusCode).toEqual(200)
     expect(JSON.parse(result.body)).toEqual({
-      ...regularUser
+      ...regularUser,
+      defaultWorkspace: expect.any(Object)
+    })
+
+    expect(result.json().defaultWorkspace).toMatchObject({
+      id: workspace.id,
```

Comment:
Ideally, we just want to check for the validity of the workspace Id, and name.

Reply:
@rajdip-b  Please check, this has been updated